sonar-db/src/main/resources/org/sonar/db/version/rows-h2.sql

```diff
@@ -510,3 +510,6 @@ INSERT INTO SCHEMA_MIGRATIONS(VERSION) VALUES ('1401');
 
 INSERT INTO USERS(ID, LOGIN, NAME, EMAIL, EXTERNAL_IDENTITY, EXTERNAL_IDENTITY_PROVIDER, USER_LOCAL, CRYPTED_PASSWORD, SALT, CREATED_AT, UPDATED_AT) VALUES (1, 'admin', 'Administrator', '', 'admin', 'sonarqube', true, 'a373a0e667abb2604c1fd571eb4ad47fe8cc0878', '48bc4b0d93179b5103fd3885ea9119498e9d161b', '1418215735482', '1418215735482');
 ALTER TABLE USERS ALTER COLUMN ID RESTART WITH 2;
+
+INSERT INTO ORGANIZATIONS (UUID, KEE, NAME, CREATED_AT, UPDATED_AT) VALUES ('AVdqnciQUUs7Zd3KPvFD', 'default-organization', 'organization.default', '1474962596482', '1474962596482');
```

Comment:
Forget, I didn't see the fixup commit (as a side note it's quite a pain to review when there are too many fixups)